verilog/CST/type.h

```diff
@@ -108,6 +139,18 @@ const verible::SyntaxTreeNode* GetParamListFromInstantiationType(
 const verible::SyntaxTreeLeaf& GetSymbolIdentifierFromEnumName(
     const verible::Symbol& enum_name);
 
+// Returns symbol identifier node for the type name from node tagged with
+// kInstantiationType (if exists) or return nullptr.
+//- e.g some_type x; return "some_type".
+const verible::SyntaxTreeLeaf* GetTypeIdentifierFromInstantiationType(
+    const verible::Symbol& instantiation_type);
+
+// Returns symbol identifier node for the type name from node tagged with
+// kDataType (if exists) or return nullptr.
+//- e.g module m(Bus x) => extracts "Bus".
```

Comment:
Remove the "module m" part from this example, it may distract/confuse the reader.  Start with a text that represents the input CST node.